test/member.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import {
+  Arc,
+  DevArcConfig as arcConfig,
+  DAO,
+  MemberData,
+  Member,
+  DAOData,
+} from "../src";
+import { render, screen } from "@testing-library/react";
+
+describe("Member component ", () => {
+  it("Shows member and dao address with inferred props", async () => {
+    const daoAddress = "0xe7a2c59e134ee81d4035ae6db2254f79308e334f";
+    const memberAddress = "0xe11ba2b4d45eaed5996cd0823791e0c93114882d";
+    const { container } = render(
+      <Arc config={arcConfig}>
+        <DAO address={daoAddress}>
+          <Member address={memberAddress}>
+            <DAO.Data>
+              <Member.Data>
+                {(dao: DAOData, member: MemberData) => (
+                  <>
+                    <div>{"Member address: " + member.address}</div>
+                    <div>{"DAO address: " + dao.id}</div>
+                  </>
+                )}
+              </Member.Data>
+            </DAO.Data>
+          </Member>
+        </DAO>
+      </Arc>
+    );
+
+    const member = await screen.findByText(/Member address:/);
+    const dao = await screen.findByText(/DAO address:/);
+    expect(member).toBeInTheDocument();
+    expect(dao).toBeInTheDocument();
+    expect(container).toMatchInlineSnapshot(`
+      <div>
+        <div>
+          Member address: ${memberAddress}
+        </div>
+        <div>
+          DAO address: ${daoAddress}
+        </div>
+      </div>
+    `);
+  });
+
+  /* 
+  This doesn't works because Member component just accept the members address
+  props, so this might change (?)
+
+  it("Shows member and dao address without inferred props", async () => {
+    const daoAddress = "0xe7a2c59e134ee81d4035ae6db2254f79308e334f";
+    const memberAddress = "0xe11ba2b4d45eaed5996cd0823791e0c93114882d";
+    const { container } = render(
+      <Arc config={arcConfig}>
+        <Member address={memberAddress} daoAddress={daoAddress}>
```

Comment:
Ahh, that's because you have to import `DAOMember`: https://github.com/dOrgTech/DAOcomponents/blob/d28c6ce11b365deff2920e0a896fd27cf722b5e5/src/components/Member.tsx#L19

Reply:
I suppose to make this more straight forward to the user, we could make the `daoAddress` prop optional on the Member class... what do you think? Is it strange to have two different Member components? DAOMember & Member?

Reply:
Yas! It's strange and maybe confuse the developer - Let's add the daoAddress optional prop into the Member class

Reply:
Okay, I agree. This is a bit of a larger change, and is closely related to the work I still need to finish in the `infer-prop` branch, so let's not tackle this now. I also can't think of a "clean" way to do this throughout the code base that's consistent. We can talk offline about why this is the case :P